src/modules/45-dashboards/pages/home/CreateDashboardForm.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState } from 'react'
+import { useParams, useHistory } from 'react-router-dom'
+import type { ModalErrorHandlerBinding } from '@wings-software/uicore'
+import { Layout, Container, useToaster } from '@wings-software/uicore'
```

Comment:
You should combine these
```suggestion
import { ModalErrorHandlerBinding, Layout, Container, useToaster } from '@harness/uicore'
```